app/controllers/v0/institution_programs_controller.rb

```diff
@@ -23,7 +23,7 @@ def autocomplete
     def index
       @meta = {
         version: @version,
-        count: search_results.count,
+        count: search_results.load.size,
```

Comment:
This makes sense because `search_results` are used in the `count_field` method.  But given my other comments, I suspect the best path would be to use `search_results.filter_count` method (as used in `institutions_controller`) instead of `count_field` in `facets`, and revert this to `search_results.count`